scripts/zkai_attention_kv_sixteen_head_fused_softmax_table_native_gate.py

```
@@ -0,0 +1,542 @@
+        if result.get(key) != value:
+            raise AttentionKvSixteenHeadFusedSoftmaxTableGateError(f"result drift for {key}")
+    validate_fused_envelope(envelope, source_input)
+    if not isinstance(mutation_results, list) or len(mutation_results) != EXPECTED_MUTATION_COUNT:
+        raise AttentionKvSixteenHeadFusedSoftmaxTableGateError("mutation result shape drift")
+    if tuple(item.get("name") for item in mutation_results if isinstance(item, dict)) != EXPECTED_MUTATION_NAMES:
+        raise AttentionKvSixteenHeadFusedSoftmaxTableGateError("mutation result name drift")
+    if any(item.get("rejected") is not True for item in mutation_results if isinstance(item, dict)):
+        raise AttentionKvSixteenHeadFusedSoftmaxTableGateError("mutation accepted")
+
+
+def build_result(envelope: dict[str, Any], source_input: dict[str, Any], mutation_results: list[dict[str, Any]]) -> dict[str, Any]:
```

Ruff (0.15.12)

[warning] Unused function argument: `source_input`

(ARG001)